### ISSUE TITLE
Download not working

### ISSUE THREAD

**Issue by MightyCoderX** [ytb2mp3/youtube-mp3-downloader#106]
```
evalmachine.<anonymous>:339
rma=function(a,b,c,d){var e=null;switch(b){case "JSON":try{var f=c.responseText}catch(h){throw d=Error("Error reading
responseText"),d.params=a,bu(d),h;}a=c.getResponseHeader("Content-Type")||"";f&&0<=a.indexOf("json")&&(")]};Yka(ncode);
                                                                                                                                                                  ^^^^^^^^^^^^^^^^

SyntaxError: Invalid or unexpected token
    at new Script (node:vm:100:7)
    at Object.exports.decipherFormats (C:\Users\matti\Documents\Programming\Web\BackEnd\Node\youtube-to-mp3-discord-bot\node_modules\ytdl-core\lib\sig.js:116:51)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async Promise.all (index 0)
    at async exports.getInfo (C:\Users\matti\Documents\Programming\Web\BackEnd\Node\youtube-to-mp3-discord-bot\node_modules\ytdl-core\lib\info.js:401:17)
```

**Comment by shaunsaker** [comment]
This [fix](https://github.com/fent/node-ytdl-core/issues/1108#issuecomment-1212705953) worked for me.

**Comment by MightyCoderX** [comment]
> This [fix](https://github.com/fent/node-ytdl-core/issues/1108#issuecomment-1212705953) worked for me.

I fixed it somehow but after a while of it working correctly it now gives me this other error
(#107)

**Comment by ytb2mp3** [comment]
This has been fixed in the `0.7.11` release.
